# Impact of Ehrlichia ruminantium infection on the tissue-specific microbiome of Amblyomma gemma ticks collected from cattle in Kenya

## Abstract

*Amblyomma gemma* ticks are principal vectors of Ehrlichia ruminantium, the causative agent of heartwater (or cowdriosis), a highly fatal ruminant disease (cattle, goat and sheep in sub-Saharan Africa (SSA). While most studies have focused on pathogen-host interaction, the effect of E. ruminantium infection on the tick's natural microbial populations, specifically at the tissue-specific level, remains poorly understood. This study aims to understand how E. ruminantium infection impacts the microbial community (composition, diversity, and inferred functional profiles) and microbial networks in midgut and salivary glands of Am. gemma collected from cattle in Kenya. We hypothesize that E. ruminantium infection will induce significant shifts in the microbiome of these two tissues, potentially influencing pathogen establishment, replication, and transmission. Using 16S rRNA gene V1-V2 regions, recently metabarcode sequenced on the Illumina platform, we will establish the bacterial communities of infected and uninfected Am. gemma midguts and salivary glands. Subsequent bioinformatic analysis will involve taxonomic classification, alpha and beta diversity testing, differential abundance testing, predictive functional profiling (e.g., with PICRUSt2), and co-occurrence network analysis to identify potential microbial interactions. The findings from this study will provide novel insights into the complex multipartite interactions among the tick, its endosymbionts, and significant pathogens, potentially revealing new targets for tick-borne disease control.


## Study Objectives

1. To characterize and compare the microbial composition and diversity of the midgut and salivary glands of Amblyomma gemma ticks infected and uninfected with Ehrlichia ruminantium.

2. To identify differentially abundant bacterial taxa associated with E. ruminantium infection in each tissue type using 16S rRNA gene sequencing data.

3. To infer functional profiles of microbial communities in infected and uninfected midgut and salivary gland tissues using predictive metagenomics (e.g., PICRUSt2).

4. To construct and analyze microbial co-occurrence networks to identify potential interactions and shifts in microbial community structure associated with E. ruminantium infection.

## References

1.	Allsopp, B. A. (2010). Natural history of Ehrlichia ruminantium. Veterinary parasitology, 167(2-4), 123-135. https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0146651
   
2.	Randall R. Jim√©nez (2021): Practical Metagenomics: Microbiome tutorial with QIIME 2. https://f1000research.com/documents/10-798/pdf
   
3.	Khogali R, Bastos A, Getange D, Bargul JL, Kalayou S, Ongeso N, Verhoeven JTP, Kabii J, Ngiela J, Masiga D, Villinger J. Exploring the microbiomes of camel ticks to infer vector competence: insights from tissue-level symbiont-pathogen relationships. Scientific Reports 2025 (Feb 15);15(1):5574. doi: 10.1038/s41598-024-81313-1.
